pkg/util/log/eventpb/telemetry.proto

```diff
@@ -71,3 +71,31 @@ message CapturedIndexUsageStats {
   bool is_unique = 10 [(gogoproto.jsontag) = ",omitempty"];
   bool is_inverted = 11 [(gogoproto.jsontag) = ",omitempty"];
 }
+
+// CreateChangefeedQuery
+message CreateChangefeedQuery {
+  CommonEventDetails common = 1 [(gogoproto.nullable) = false, (gogoproto.jsontag) = "", (gogoproto.embed) = true];
+  CommonJobEventDetails job = 2 [(gogoproto.nullable) = false, (gogoproto.jsontag) = "", (gogoproto.embed) = true];
+
+  // The number of tables listed in the query that the changefeed is to run on.
+  int32 num_tables = 3 [(gogoproto.jsontag) = ",omitempty"];
+
+  // The type of sink being emitted to (ex: kafka, nodelocal, webhook-https).
+  string sink_type = 4 [(gogoproto.jsontag) = ",omitempty", (gogoproto.moretags) = "redact:\"nonsensitive\""];
+
+  // The behavior of emitted resolved spans.
```

Comment:
is this the value of resolved option (like "resolved=10s")?  Is it useful? Or do we care more about the fact that resolved option was specified?
Perhaps we can include a boolean? And that's sufficient?

Also, should we perhaps include booleans for things like initial_scan? I know we have plenty of various options now for this (including the recently added initial_scan_only)... But maybe we can have initial_scan as a string which can take one of the values:
   off, only, cursor (i.e. no_initial_scan was specified, so cursor was either specified or we used statement time).